frontend/src/metabase/query_builder/components/view/ViewHeader/components/SavedQuestionLeftSide/SavedQuestionLeftSide.1.tsx

```diff
@@ -0,0 +1,121 @@
+import cx from "classnames";
```

Comment:
Is this `SavedQuestionLeftSide.1.tsx` file a leftover?

Reply:
removed